typescript/README.md

```diff
@@ -1,6 +1,21 @@
-# General Style Standards
```

Comment:
Can you move the old contents of this file into a separate markdown file? Maybe `style-guide.md`?

I feel like it’s unrelated (and distracting) for the rest of the contents now